File: packages/sdks/javascript/src/services/CredentialService.ts
import { CredentialsApiService } from "../api/generated/apis/credentials-api";
import { CredentialsRepositoriesApiService } from "../api/generated/apis/credentials-repositories-api";

export class CredentialService extends CredentialsApiService {
    public repositories = new CredentialsRepositoriesApiService();
}

File: packages/sdks/python/src/devopness/services/credential_service.py
"""
Devopness API Python SDK - Painless essential DevOps to everyone
"""

from ..generated.api.credentials_api import (
    CredentialsApiService,
    CredentialsApiServiceAsync,
)
from ..generated.api.credentials_repositories_api import (
    CredentialsRepositoriesApiService,
    CredentialsRepositoriesApiServiceAsync,
)

__all__ = ["CredentialService", "CredentialServiceAsync"]


class CredentialService(
    CredentialsApiService,
    CredentialsRepositoriesApiService,
):
    """Service for credentials in the Devopness API."""


class CredentialServiceAsync(
    CredentialsApiServiceAsync,
    CredentialsRepositoriesApiServiceAsync,
):
    """Async service for credentials in the Devopness API."""
